sphinx/tutorial/cylc/furthertopics/queues.rst

```diff
@@ -89,21 +78,20 @@ Replace the ``[scheduling]`` line with:
 
 Make sure you are in the root directory of your suite.
```

Comment:
I would change 'root' to '``root``' if you get a chance, just to be consistent in terms of formatting.